.github/workflows/deploy.yml

```diff
@@ -301,22 +301,22 @@ jobs:
               return "prod"
```

Comment:
@daveb-hni  can we please add new environments here: trn and vc1, rather than creating new branch just to add those envs